src/views/Nft/market/Collection/IndividualNFTPage/index.tsx

```diff
@@ -45,6 +53,19 @@ const IndividualNFTPage = () => {
 
   const isPBCollection = collectionAddress.toLowerCase() === pancakeBunniesAddress.toLowerCase()
 
+  useUpdateNftInfo(collectionAddress)
+
+  useEffect(() => {
+    // Fetch first 30 NFTs on page load
+    // And then query every 10 sec in case some new (cheaper) NFTs were listed
+    const msSinceLastUpdate = Date.now() - latestFetchAt
+    // Check for last update is here to prevent too many request due to fetchMorePancakeBunnies updating too often
+    // (it can't be reasonably wrapper in useCallback because the tokens are updated every time you call it, which is the whole point)
+    if (msSinceLastUpdate > 10000 && !isFetchingMoreNfts) {
```

Comment:
Similar thing as above - can't memoize it well enough not to retrigger effect all the time.